blazor/pager/behaviour-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Behaviour Settings in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about Behaviour Settings in the Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Behaviour Settings in Pager Component
+
+## Page size
+
+The `PageSize` property of the Pager control allows an option to get or set a value that indicates whether to define the number of records displayed per page.
```

Comment:
through out the document mention as **Pager component** instead of Pager control. For blazor platform we need to mention as component only